src/main/java/com/greenfox/lvlup/controller/LibraryTestController.java

```diff
@@ -0,0 +1,27 @@
+package com.greenfox.lvlup.controller;
+
+import com.greenfox.lvlup.model.entityTestingDto.BadgeDTO;
+import com.greenfox.lvlup.service.BadgeService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+import java.util.List;
+
+@RestController
+public class LibraryTestController {
+    private BadgeService badgeService;
+
+    @Autowired
+    public LibraryTestController(BadgeService badgeService) {
+        this.badgeService = badgeService;
+    }
+
+    @GetMapping("/library")
+    public List<BadgeDTO> getAllDtos() {
```

Comment:
smaller than minor thing: I'd suggest that we should be a bit more consistent about the spelling of DTO, e.g. let's have 'dto' as the package name and use 'DTO' everywhere else (instead of e.g. 'Dto' in this method's name). Let's have the others' input, too, ofc :)

Reply:
Changed the function name to getAllDTOs:

@GetMapping("/library")
    public List<BadgeDTO> getAllDTOs() {
        return badgeService.getDTOListFromBadge();
    }